backend/utils/codeValidation.js

```diff
@@ -0,0 +1,25 @@
+const ensureLegalCode = (req, res, next) => {
+  const { code } = req.body;
+  const blackList = ['throw', 'Error'];
+  const splitCode = code.split(' ');
+  let legalCode = true;
+  for (let i = 0; i < blackList.length; i += 1) {
+    if (splitCode.includes(blackList[i])) {
+      legalCode = false;
+      res.status(200).send({
```

Comment:
We should prob send a `400` instead. With an error explaining that the user is using prohibited keywords.